File: Manager/src/main/java/com/xxx/proj/controller/IndexController.java
package com.xxx.proj.controller;

import org.springframework.security.core.context.SecurityContextHolder;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

import java.util.HashMap;
import java.util.Map;

@RestController
@RequestMapping("/index")
public class IndexController {

    @RequestMapping("/getName")
    public Map getName() {
        String username = SecurityContextHolder.getContext().getAuthentication().getName();
        Map<String, String> map = new HashMap<>();
        map.put("loginName", username);
        return map;
    }
}

File: Manager/src/main/webapp/js/controller/indexController.js
//控制层
app.controller('indexController', function ($scope, $controller, indexService) {

    $controller('baseController', {$scope: $scope});//继承

    //读取列表数据绑定到表单中  
    $scope.getName = function () {
        indexService.getName().success(
            function (response) {
                $scope.loginName = response.loginName;
            }
        );
    }


});	